mongo/integration/index_view_test.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 type index struct {
-	Key  map[string]int
```

Comment:
Our tests were using a non-order preserving type for key specs, so I updated this.